source/_docs/guides/circleci.md

```diff
@@ -0,0 +1,235 @@
+        </tr>
+        <tr>
+          <td><code>TEST_SITE_NAME</code></td>
+          <td>Used to set the name of the test site when installing Drupal.</td>
+        </tr>
+        <tr>
+          <td><code>ADMIN_EMAIL</code></td>
+          <td>Used to configure the email address to use when installing Drupal.</td>
+        </tr>
+        <tr>
+          <td><code>ADMIN_PASSWORD</code></td>
+          <td>Used to set the password for the UID 1 user during site installation.</td>
+        </tr>
+        <tr>
+          <td><code>GIT_EMAIL</code></td>
+          <td>Used to configure the Git user’s email address for commits we make.</td>
+        </tr>
+        <tr>
+          <td><code>GITHUB_TOKEN</code></td>
+          <td>Used by CircleCI to post comments on pull requests.</td>
+        </tr>
+      </tbody>
+    </table>
+
+    Now you can rerun the failed job again from the **Builds** tab in CircleCI and follow the build from the dashboard. This time it should pass.
+
+    ![Running tests](/source/docs/assets/images/integrations/running.png)
+
+## Test the New Workflow
+Now make a change and submit a pull request to take your new workflow for a spin.
+
+1. From the root directory of your local project, create a new branch and open a new file:
+
+    ``` bash
+    git checkout -b new-feature
+    vim tests/features/content-ui.feature
+    ```
+
+    If you're unfamiliar with `vim`, you can use another text editor to create the new file.
+
+
+2. Add the following:
+
+    ``` ruby
+    Feature: Create Content through Drupal Content UI
+      In order to know that the Drupal content UI is working
+      As a website user
+      I need to be able to add a basic page
+
+      @api
+      Scenario: Add a basic page
+        Given I am logged in as a user with the "administrator" role
+        And I am on "/node/add/page"
+        And I enter "Test Page" for "Title"
+        And I press "Save and publish"
+        Then I should see "Basic page Test Page has been created."
+    ```
+
+3. Commit your change, and push your new branch to GitHub:
+
+    ```
+    git add .
+    git commit -m "new feature"
+    git push origin new-feature
+    ```
+
+4. From your GitHub repo, go to the new branch and open a Pull Request:
+
+    ![New PR](/source/docs/assets/images/integrations/new_pr.png)
+
+    Commits pushed to GitHub will initiate a new build with CircleCI. This is a simple change, so all tests should pass. The tests run on GitHub and then are pushed to a Multidev environment on Pantheon.
+
+If you are satisfied with the change, merge your pull request. This will cause another test to run on CircleCI against the master branch. Once these tests pass, your pull request will be merged and the changes will be available on your development site. Use the [Pantheon workflow](/docs/pantheon-workflow/) to merge into Dev and deploy to Test then up to Live, either with Terminus or on the Site Dashboard.
+
+## Conclusion
+
```

Comment:
Also suggest we point folks to Greg's ["Update Your Project"](https://pantheon.io/docs/guides/github-pull-requests/#update-your-project) section for how to manage updates - maybe also note that one-click updates in the Dashboard can't (or shouldn't?) be used in this workflow.